wordpress-mobile/WordPress-iOS
# Respect API ordering for page template categories

This PR adds an ordinal attribute to the `PageTemplateCategory` data model to explicitly use the API ordering while caching results from the remote fetch.

To test:

* Add a new page (from the my sites tab or page list)
* Observe the order of the categories in the filter bar and sections
  * The ordering should reflect the API order (e.g. in my case it was Blog, Highlights, Splash, About, Links, Home, Team, Services, Portfolio, Contact, Gallery, Featured, but generally it should not be strictly alphabetical).

PR submission checklist:
